server/bundler/hot-reloader.js

```diff
@@ -4,6 +4,8 @@
  * External dependencies
  */
 var socketio = require( 'socket.io' );
+var debug = require( 'debug')( 'hot-reloader' );
```

Comment:
@retrofox Thanks, I've added the prefix using the convention I've seen in client/, however in server/ I cannot use ES6 so I've kept the plain require( 'debug' )( ... ).